docs/wiki/deployment/debugging.md

```diff
@@ -125,6 +125,11 @@ While not expected, the backing store may be corrupted by problems with the file
 
 If your `--database_path` is `/var/osquery/osquery.db` then the backup is `/var/osquery/osquery.db.backup`. The database is always a folder and the backup location is the suffix ".backup" appended.
 
+### No query solution errors
+
+You may see `no query solution` errors in status logs, as the result of an `osqueryi` query, etc.
+Tables that have a required column are documented on [the schema page](https://osquery.io/schema/).
```

Comment:
cut and paste error?
```suggestion
This typically means a REQUIRED columns must be present in the WHERE clause. For example, the `file` table needs either a `directory` or `path`.
See the [the schema page](https://osquery.io/schema/) for table required column documentation.
```

cut and paste error? (and rewording a sentence)

Reply:
Actually, missing REQUIRED columns will not give this error anymore. :)

Reply:
I might just remove this section.

Reply:
Removal seems fine.